dpytools/embeds.py

```diff
@@ -118,3 +119,141 @@ def add_fields(self, inline=True, **kwargs):
         """
         for name, value in kwargs.items():
             self.add_field(name=name, value=value, inline=inline)
+
+            
+class EmbedList:
+    
+    """A class that takes dictionary containing name and value and splint them into several Embeds.
```

Comment:
If this class only really focuses on paginating fields, perhaps its better to change its name and description to that.